### ISSUE TITLE
Setting of column width with css does not work

### ISSUE THREAD

**Issue by Azador** [TobiasBg/TablePress#214]
I've tried to change the width of some columns in a table, but the entered css code is not accepted. It is overwritten with inline css at the element "col" inside the element "colgroup".
I guess this is added by a script, because these elements does not exist if I look at the source of the page, it only shows up in the page inspector.

CSS used: `.tablepress-id-1 .column-4 { width: 80px; }`
Relevant part of the page source:
```html
<table id="tablepress-1" class="tablepress tablepress-id-1">
<caption style="caption-side:bottom;text-align:left;border:none;background:none;margin:0;padding:0;"><a href="[https://www..." rel="nofollow">Bearbeiten</a></caption>
<thead>
<tr class="row-1">
	<th class="column-1">C1</th><th class="column-2">C2d</th><th class="column-3">C3</th><th class="column-4">Column 4 with a longer header</th><th class="column-5">C5</th><th class="column-6">C6</th><th class="column-7">C7</th><th class="column-8">C8</th><th class="column-9">C9</th><th class="column-10">C10</th><th class="column-11">C11</th><th class="column-12">C12</th><th class="column-13">C13</th>
</tr>
</thead>
<tbody class="row-striping row-hover">
<tr class="row-2">
	<td class="column-1">Data1</td>
``` 

In the inspector, I see two tables. One for the header inside `div.dt-scroll-head` and one for the data inside `div.dt-scroll-body`. In both exists the element <colgroup> to define the width of the columns, preventing the width in the CSS code to take effect.
The table has 168 rows (including the header) and 13 columns.

Maybe I've been missing something but I followed the instructions at the FAQ in section [How can I set column widths?](https://tablepress.org/faq/column-widths/).

**Comment by TobiasBg** [comment]
Hi @Azador,

indeed, when the "Table Features for Site Visitors" are used, that `<colgroup>` is added dynamically via JavaScript.
However, it should also take into account the widths that are defined with CSS code.
Can you please post me a link to the page with the table where this problem happens, so that I can take a direct look? Thanks!

Best wishes,
Tobias

**Comment by Azador** [comment]
Hi Tobias,

so, that was my fault:
I wanted to reduce the width of a column further than the header entry allowed.
In the customizer of Wordpress, the width did not change in the live preview due to the fact, that the JavaScript was already run and the inline css of the colgroup took precedence, so I couldn't see any changes, even if increasing the size of the column.
I've also tried the changed width with the normal page view and reloaded it, but in that case, I only tried with smaller column sizes for that was the effect, I wanted to achieve. But, as the documentation states, that is not possible. I'm afraid, I have to change the contents to change the width of the columns.

Many thanks, for your fast response and the hint with the script code, that takes the CSS into account.
Sorry, that I didn't check that in advance.

Greetings,
Azador
